Client/Application/AppDelegate+SyncSentTabs.swift

```diff
@@ -63,7 +63,9 @@ class AppSyncDelegate: SyncDelegate {
                 let trigger = UNTimeIntervalNotificationTrigger(timeInterval: 0.1, repeats: false)
 
                 // The identifier for each notification request must be unique in order to be created
-                let requestIdentifier = "\(SentTabAction.TabSendCategory).\(url.absoluteString)"
+                // we attach a unique identifier in case the user sends the tab multiple times.
+                let uuidString = UUID()
+                let requestIdentifier = "\(uuidString).\(SentTabAction.TabSendCategory).\(url.absoluteString)"
```

Comment:
Are you confident the displaySentTab function is always called just once in response to a sent tab?
This could get kind of spammy if it's called multiple times.
I worry the purpose of this identifier originally being constructed this way instead of just any random value was to intentionally make the request idempotent.
If you haven't already I'd suggest doing some testing just to be sure.